src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.FeatureCollection.cs

```diff
@@ -182,7 +182,7 @@ CancellationToken IHttpRequestLifetimeFeature.RequestAborted
 
         bool IHttpResponseFeature.HasStarted => HasResponseStarted;
 
-        bool IHttpUpgradeFeature.IsUpgradableRequest => _server.IsWebSocketAvailable(_pInProcessHandler);
+        bool IHttpUpgradeFeature.IsUpgradableRequest => true;
```

Comment:
Don't we still need this check?

Reply:
We reset feature to null if websockets are disabled (https://github.com/aspnet/IISIntegration/blob/3bdb33ccd4dbe5baa3162ca3d3778df1883db093/src/Microsoft.AspNetCore.Server.IIS/Core/IISHttpContext.cs#L151) so if user was able to get the feature request is upgradeable.